tensorflow/python/keras/distribute/distribute_strategy_test.py

```diff
@@ -740,6 +748,28 @@ def test_numpy_with_sample_weights(self, distribution, cloning):
       model.fit(inputs, targets, sample_weight=sample_weights, epochs=1,
                 steps_per_epoch=2, verbose=1)
 
+  @combinations.generate(
+      combinations.combine(distribution=strategies_minus_tpu, mode=['graph']))
```

Comment:
Any reason we did not use "distribution=strategy_minus_tpu_combinations" to test "graph" and "eager"?

Reply:
If we run into issues while testing mode="eager" we can look into it in a followup PR. For now it will be good to submit what we have.